readme.md

```diff
@@ -198,6 +198,10 @@ Since the bulk of the config options focus around connecting to the various, sup
 not all config values apply for each type of database being used. The following chart attempts
 to define each option based on the database type being used.
 
+**Note:** If `db_name` is set, schema tool will attempt to connect to the DB listed on `init`.
+Therefore it is best practice to create that DB before initializing schema tool or running your alters.
+You should not create or destroy that DB as part of your alter chain.
+
```

Comment:
If I'm reading the "Configuration" table in the Readme correctly, `db_name` is only relevant for Postgres or Vertica. So this does exclude MySQL because you wouldn't define the `db_name` key if you were using MySQL.